tests/certification/bindings/azure/eventhubs/components/binding/iothub/eventhubs.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: azure-eventhubs-binding
+  namespace: default
+spec:
+  type: bindings.azure.eventhubs
+  version: v1
+  metadata:
+    - name: enableEntityManagement
+      value : true
+    - name: connectionString
+      secretKeyRef:
+        name: AzureIotHubEventHubConnectionString
+        key: AzureIotHubEventHubConnectionString 
+    - name: credentials
+      secretKeyRef:
+        name: AZURE_CREDENTIALS
```

Comment:
This is not the right value.

The connection string should come from secret `AzureIotHubEventHubConnectionString`